### PR TITLE
Docs added for dark theme

### LINKED ISSUE #922: [Docs] Update contributor guide with enhanced with how dark mode works
**Current State:**
Meshery.io has a new dark mode.

**Desired State:**
New and existing contributors need training / documentation on how this feature works. 

#### Acceptance Criteria
1. Concise instructions and example(s) should be included in this repo's readme.
2. A similar set of instructions should be included in a new page in https://docs.meshery.io/project/contributing

---
**Contributor Resources**
- [Meshery documentation site](https://docs.meshery.io)
- [Meshery documentation source](https://github.com/meshery/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by Shivam-AfA**
**Description**

This PR fixes #922 

**Notes for Reviewers**
This PR adds the documentation for new contributers to understand how they can contribute to Meshery.io website based on light and dark theme features on the website.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: images/Kuma.svg [hunk 1]
```diff
@@ -1 +1 @@
...iewBox="0 0 400 400" xml:space="preserve" style="enable-background:new 0 0 400 400"><style type="text/css">.st0{fill:#290b53}</style><g><path d="M1073,3340c-145-31-239-134-263-287c-18-118,17-213,140-374l58-76l-18-54c-139-406-150-444-150-496 c0-117,57-257,169-413c73-103,168-202,456-476c121-115,291-278,378-362c87-83,162-152,167-152s87,73,182,162s196,182,223,207 c485,442,622,591,713,775c48,96,61,147,62,236c0,56-11,97-81,295c-45,127-88,253-96,282l-15,51l45,64c101,143,128,227,108,336 c-16,87-43,142-97,197c-65,64-134,88-259,89c-117,1-179-17-250-71l-46-35l-127,30c-152,37-177,41-191,32c-8-4,2-58,29-166 c68-265,158-614,180-694c37-136,42-173,38-246c-8-128-83-239-196-292c-52-25-69-27-207-30c-134-3-157-1-207,18 c-142,53-241,214-225,365c4,43,112,484,230,942c13,51,22,95,19,98c-17,17-160-6-314-51l-57-17l-38,33c-62,55-128,79-229,85 C1155,3347,1096,3345,1073,3340z" class="st0" transform="translate(0.000000,400.000000) scale(0.100000,-0.100000)"/><path d="M1862,2540c-12-5-25-17-29-25c-9-24,24-294,38-311c9-11,41-14,140-14c122,0,129,1,143,23c15,24,40,275,29,302 c-3,9-17,21-30,26C2122,2553,1892,2552,1862,2540z" class="st0" transform="translate(0.000000,400.000000) scale(0.100000,-0.100000)"/></g></svg>
```
Comment (Shivam-AfA) [inline images/Kuma.svg:1]: Ignore please.

**Review by leecalcote — DISMISSED** [review]
LGTM

**Review by GaganpreetKaurKalsi — CHANGES_REQUESTED** [review]
A spelling error 
image

**Comment by Shivam-AfA** [comment]
> A spelling error image

Corrected. @GaganpreetKaurKalsi

**Review by GaganpreetKaurKalsi — APPROVED** [review]
All good 💯

**Review by eeshaanSA — APPROVED** [review]
Looks Neat. :+1:

**Comment by Shivam-AfA** [comment]
@leecalcote I think this PR is all set to be merged.

**Comment by leecalcote** [comment]
Good deal. 👏

**Pull request merged**